### ISSUE TITLE
Accessibility: Wide Glyph Detection

### ISSUE THREAD

**Issue by cinnamon-msft** [microsoft/terminal#1354]
# Summary of the new feature/enhancement

Currently, wide glyphs can be read and accessed properly by a narrator. However, when expanding the text range unit-by-unit, the text range only sees half of the wide glyph, and fails to understand it.

# Proposed technical implementation details (optional)

- [ ] Modify existing fundamental accessibility model to detect wide glyphs and read them properly when encountered
